examples/heat_equation/CMakeLists.txt

```diff
@@ -0,0 +1,121 @@
+##
+# CMake script for large project, with one executable
+# and a library of objects, optionally
+# depending on dealii-sak. 
+#
+# The structure of the directory is assumed to be: 
+# ./source: containing implementations and one file with "int main()"
+# ./include: containing all class declarations
+# ./tests: containing pairs of files: test.cc/test.output to test
+#
+# If you set the environemnt variable DEAL_II_SAK, 
+# everything will work out of the box
+##
+
+# Set the name of the project and target
+# If your application follows the structure above, you don't need to 
+# specify anything else. 
+SET(TARGET heat_equation)
+SET(_main source/main.cc)
+
+# Set the _main variable to empty if you don't want an executable
+# but only a library
+#SET(_main "")
+
+############################################################
+# Normally you shouldn't need to change anything below.
+############################################################
+# Declare all source files the target consists of:
+file(GLOB _files source/*cc)
+
+
+# Don't compile the main file into the library
+IF(NOT "${_main}" STREQUAL "") 
+    LIST(REMOVE_ITEM _files
+      ${CMAKE_CURRENT_SOURCE_DIR}/${_main}
+    )
+ENDIF()
+
+INCLUDE_DIRECTORIES(include)
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
+
+FIND_PACKAGE(deal.II 8.3 REQUIRED
+  HINTS ${deal.II_DIR} ${DEAL_II_DIR} ../ ../../ $ENV{DEAL_II_DIR}
+)
+
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic")
+DEAL_II_INITIALIZE_CACHED_VARIABLES()
+PROJECT(${TARGET})
+
+
+SET(DEAL_II_SAK "$ENV{DEAL_II_SAK}" CACHE PATH "Path to dealii-sak")
+
```

Comment:
Add an option for the testing:

~~~
OPTION(ENABLE_TESTING "Enable testing for this target." OFF)
~~~